html-skeleton/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML SKeletons</title>
+<body>Hell, HTML!</body>
+</head>
```

Comment:
Hi Quan, try moving the `<body>` element below the `<head>` element. They should be siblings 

```html
<head>
  <title></title>
</head>
<body>
</body>
```